doc/manuals/admin/build_source.md

```diff
@@ -93,38 +93,43 @@ The Orion Context Broker comes with a suite of unit, valgrind and end-to-end tes
 
 * Install Google Test/Mock from sources. Previously the URL was http://googlemock.googlecode.com/files/gmock-1.5.0.tar.bz2 but Google removed that package in late August 2016 and it is no longer working.
 
-        sudo apt-get install python2
         wget https://nexus.lab.fiware.org/repository/raw/public/storage/gmock-1.5.0.tar.bz2
         tar xfvj gmock-1.5.0.tar.bz2
         cd gmock-1.5.0
         ./configure
-        sed -i 's/env python/env python2/' gtest/scripts/fuse_gtest_files.py  # little hack to make installation to work on Debian 11
+        # Adjust /path/to/fiware-orion in the next line accordingly to where you local copy of fiware-orion repo is in your system
```

Comment:
Probably describing what this patch do / why is needed is a good idea

Reply:
It's done in the .patch file itself:

> This patch adapt the original fuse_gest_files.py to Python 3
> Check build_source.md file to see how it is used

Reply:
OK, NTC